src/pages/Character/Character.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from "react";
+import EpisodeCard from "../../components/EpisodeCard";
+import CharacterCard from "../../components/CharacterCard";
+
+import { getCharacter, getUrl } from "../../api";
+
+import Layout from "../../components/Layout";
+
+const makePromise = (urls = []) => {
+  return urls.map((url) => getUrl(url));
+};
+
+class Character extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      character: null,
+      episodes: [],
+      hasLoaded: false,
+      hasError: false,
+      errorMessage: null,
+    };
+
+    this.loadCharacter = this.loadCharacter.bind(this);
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
+    const { characterId } = match.params;
+    this.loadCharacter(characterId);
+  }
+
+  async loadCharacter(characterId) {
+    try {
+      const { data } = await getCharacter(characterId);
+      // eslint-disable-next-line compat/compat
+      const episodeResponse = await Promise.all(makePromise(data.episode));
+      const episodes = episodeResponse.map((episode) => episode.data);
+      this.setState({
+        character: data,
+        episodes: episodes,
+        hasLoaded: true,
+      });
+    } catch (error) {
+      this.setState({
+        hasLoaded: true,
+        hasError: true,
+        errorMessage: error.message,
+      });
+    }
+  }
+
+  render() {
+    const {
+      hasLoaded,
+      hasError,
+      episodes,
+      character,
+      errorMessage,
+    } = this.state;
+    return (
+      <Layout>
+        <section className="row">
+          {hasLoaded && !hasError && (
+            <CharacterCard
```

Comment:
It's OK to use the character card but in the readme we needed to add more information to this page like location and origin separately.